ofrak_type/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to `ofrak-type` will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/redballoonsecurity/ofrak/tree/master)
+
+## [2.1.0](https://github.com/redballoonsecurity/ofrak/releases/tag/ofrak-type-v2.1.0) - 2023-01-20
```

Comment:
One could make an argument that PatchMaker should be 3.0.0, given the removal of `ToolchainVersion` and updated constructor for `PatchMaker`. `ofrak` setup.py would then need to be updated.